### ISSUE TITLE
Double devtools message

### ISSUE THREAD

**Issue by suinly** [FiguredLimited/vue-mc#120]
Hi,

When I import model I have a double message by devtools (chrome) in browser console:

```
admin.js:123789
You are running Vue in development mode.
Make sure to turn on production mode when deploying for production.
See more tips at https://vuejs.org/guide/deployment.html

admin.js:105945
You are running Vue in development mode.
Make sure to turn on production mode when deploying for production.
See more tips at https://vuejs.org/guide/deployment.html
```

Sometimes it make crash vue-devtools extension. Without import vue-mc models message print once and extension works fine.

**Comment by mrFANRA** [comment]
Hi! Are you use NuxtJS?

**Comment by suinly** [comment]
No. I use laravel-mix, vuejs 2.6.10, vue-mc 0.5.0.

**Comment by mrFANRA** [comment]
I have some trouble when using vue-mc only with NuxtJS ...

**Comment by petitkriket** [comment]
I have the same error, could it be linked to the error reported at #121
vue-mc needs  2.5.17 instead of 2.*

it causes Vue to be loaded twice as seen in webpack-bundle-analyzer
![Screenshot_20190731_144810](https://user-images.githubusercontent.com/1753019/62213069-69a4e980-b3a2-11e9-9f5d-500589852035.png)

how to circumvent that ?

**Comment by suinly** [comment]
> I have the same error, could it be linked to the error reported at #121

Thanks, I checked your version.

I replaced `"vue": "2.5.17"` with `"vue": "^2.5.17"` in `node_modules/vue-mc/package.json`, deleted the directory `node_modules/vue-mc/node_modules/vue/` and rebuilded my project. After that, the vue stopped loading twice.

**Comment by petitkriket** [comment]
@suinly it fixes the problem only on your local developpment environment right ?

**Comment by petitkriket** [comment]
A PR has been opened [here](https://github.com/FiguredLimited/vue-mc/pull/122) :tada:

**Comment by rtheunissen** [comment]
This has been fixed and released as part of [v0.6.0](https://github.com/FiguredLimited/vue-mc/releases/tag/v0.6.0).
